Tools/environment_install/install-ROS-ubuntu.sh

```diff
@@ -246,6 +247,27 @@ else
     echo "Skipping adding ardupilot_ws to your home folder."
 fi
 
+
+if maybe_prompt_user "Add ardupilot_gazebo to your home folder [N/y]?" ; then
+    if [ ! -d $AP_GZ_ROOT ]; then
+        sudo apt install libgz-sim7-dev rapidjson-dev
```

Comment:
`sudo apt install gz-garden rapidjson-dev` is needed to install all the libraries. See https://gazebosim.org/docs/garden/install_ubuntu.